### PR TITLE
Update UI Theme

### PR DESCRIPTION
**Pull request by thayashi**
Basically this is UI theme update, also this includes css, templates clean ups and API test widget.
I intentionally didn't include "bootstrap-dialog.css", because default css is just enough.

### CODE REVIEW

**Comment by thayashi** [comment]
It look like:
screen shot 2016-11-16 at 7 24 48 pm

File: app/js/views/topView.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+   * @override View.constructor
+   * @param {Object} options
+   */
+  constructor(options) {
+    super(options);
+    this.app = options.app;
+    this.errorView = new this.app.ErrorClass();
+
+    console.log(this.app);
+    console.log('VERSION:', VERSION);
```
Comment (stepek) [inline app/js/views/topView.js]: Please remove this console.logs

**Comment by thayashi** [comment]
Removed

**Comment by stepek** [comment]
LGTM

**Pull request merged**